packages/solarwinds-apm/src/init.ts

```diff
@@ -229,3 +258,9 @@ export function oboeLevelToOtelLogger(
       return logger.verbose.bind(logger)
   }
 }
+
```

Comment:
How about a comment on why this `grpcMetadata` is needed? I'ms assuming it's because setting `OTEL_*HEADERS` env var doesn't work--is it similar to https://github.com/open-telemetry/opentelemetry-js/discussions/3195?

Reply:
It's actually not really needed, was too focused on exporting to our own collector.